src/app/settings/page.tsx

```diff
@@ -0,0 +1,117 @@
+import Image from "next/image";
+import React from 'react';
+import style from "./settings.module.css";
+import tennis from "../settings/images/tennispic.png";
+import profile from "../settings/images/Vector.png";
+import "./images/evaEmailFill2.png";
+import Children from "./addremove";
+
+export default function Settings() {
+    console.log("Current working directory:", process.cwd());
+
+    return (
+        <div className={style.body}>
+            <div className={style.allDivs}>
+                <form>
+                    <div className={style.top}>
+                        <div className={style.accHead}>
+                            <h3>Account</h3>
+                        </div>
+                        <Image src={tennis} alt="tennis"/>
+                    </div>
+                    
+                    <br/>
+                    <hr className={style.hhhr}></hr>
+                    <br/>
+
+                    <Image src={profile} alt="profile" priority={false}/>
+                    <br/>
+                    <br/>
+                    <h3>Full Name</h3>
+                    <br/>
+
+                    <div className={style.textNames}>
+                        <div className={style.nameTitles}>
+                            <label htmlFor='name' > First Name </label> <br/>
+                            <input type="text" id="first_name" placeholder='' className={style.individualNames} required/>
+                        </div>
+                        <div className={style.nameTitles}>
+                            <label htmlFor='name' > Last Name </label> <br/>
+                            <input type="text" id="last_name" placeholder=''   className={style.individualNames} required/>
+                        </div>
+                    </div>  
+                    <br/>
+                    <hr className={style.hhhr}></hr>
+                    <br/>
+                    <h3>Personal Info</h3>
+                    <br/>
+
+                    <div className={style.personal}>
+                        <label htmlFor='name'> Email </label> <br/>
+                        <input type="text" id="email" className={style.textbox}/>
+                        <br></br>
+                        <div className={style.textNames}>
```

Comment:
There should be some spacing between phone number and zip code. The input boxes are not the same size in the Figma, but it might look better if they were.

Reply:
So I should definitely make them the same size and ignore the Figma on this one?

Reply:
yea